open-api/rest-catalog-open-api.yaml

```diff
@@ -2106,6 +2204,28 @@ components:
           items:
             $ref: '#/components/schemas/PartitionStatisticsFile'
 
+    PlanTask:
+      description:
+        An opaque JSON object that contains information provided by the REST server,
+        to be utilized by clients for distributed table scan planning, should be supplied
+        as is for input in `PlanTable` operation.
+      type: object
+
+    FileScanTask:
+      type: object
+      required:
+        - data-file
+      properties:
+        data-file:
+          $ref: '#/components/schemas/ContentFile'
```

Comment:
`ContentFile` should not be used directly. This should be `DataFile`.

Similarly, `delete-files` below should be a list of either `PositionDeleteFile` or `EqualityDeleteFile`.

Reply:
will fix

Reply:
will fix